[FEAT] Batch's jobs are now returned in the same order they were created (#99)
pasqal-io/pasqal-cloud: CHANGELOG.md

All notable changes to this project will be documented in this file.

## [0.3.4] - 2023-07-21

### Added

> Now jobs are ordered by creation date in a list through the attribute `ordered_jobs` when manipulating a Batch object.

Reviewer: ```suggestion
Added an `ordered_jobs` list as an attribute to the `Batch` object in which jobs are ordered by creation date.
```

Author: I added a comment about jobs being deprecated from now

All notable changes to this project will be documented in this file.

## [0.3.4] - 2023-07-21

### Added

> Added an `ordered_jobs` list as an attribute to the `Batch` object in which jobs are ordered by creation date.

Reviewer 2: Aren't Python dicts ordered in newer pythons (after 3.6 or something like that)?
So we could just keep the `jobs`? or does the db now return a list?

Author: The db return a list of Jobs
